runtime/class_instance_decl.inl

```diff
@@ -33,6 +33,7 @@ public:
   class_instance() = default;
   class_instance(const class_instance &) = default;
   class_instance(class_instance &&) noexcept = default;
+  class_instance(T *ptr) noexcept : o(ptr, true) {}
```

Comment:
Is it possible to get rid of this constructor and use `class_instance<T>::create_from_base_raw_ptr`?

Reply:
Yes, sure, my fault